build-tools/packages/build-tools/src/common/fluidUtils.ts

```diff
@@ -165,6 +151,18 @@ export function getFluidBuildConfig(rootDir: string, noCache = false): IFluidBui
 		configExplorer.clearCaches();
 	}
 
-	const config = configExplorer.search(rootDir);
-	return config?.config;
+	const configResult = configExplorer.search(rootDir);
+	const config = configResult?.config as IFluidBuildConfig | undefined;
+
+	if (config === undefined) {
+		throw new Error("No fluidBuild configuration found.");
+	}
+
+	// Only version 1 of the config is supported. If any other value is provided, throw an error.
+	if (config?.version !== FLUIDBUILD_CONFIG_VERSION) {
```

Comment:
Should you allow version to be "undefined"?

Reply:
Hmmm, I guess the "safe" thing to do is make it nullable and coerce it to a default value, but I'd like to make it required in a future release. For now I'll warn if it's undefined and error if it's an unsupported version.